src/k8s/go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/canonical/k8s
 go 1.22.5
 
 require (
-	github.com/canonical/go-dqlite v1.21.0
-	github.com/canonical/lxd v0.0.0-20240709081608-9df996e36bac
-	github.com/canonical/microcluster v0.0.0-20240725000112-da36ecbd845e
+	github.com/canonical/go-dqlite v1.22.0
+	github.com/canonical/lxd v0.0.0-20240730172021-8e39e5d4f55f
```

Comment:
it's a bit weird that an LTS microcluster relies on an unversioned lxd commit?

Reply:
According to the microcluster team:
```
LXD itself doesn't use a Go compatible tag versions, because history, and also because if we were to switch to using it them (which we considered and tried just before the last LTS) we'd have to bump the Go mod version to match that would be pretty disruptive as all Go imports would need to change.

For the client and api packages these could potentially be moved into a separate repo, this too we have considered, and not ruled out, but that too would be pretty disruptive and we didn't want to make any more large changes when the license, version numbering scheme and organisation umbrella were changing at the same time :slightly_smiling_face:

One thing that is also worth keeping in mind with the LXD client and api package is that we try and keep even our LTS branches in sync with main because we expect older clients to be able to talk to newer servers and vice versa, so try pretty hard to not break compatibility at that level, and introduce new features via API extensions and new fields.

So although its an unversioned commit, that is effectively the current released version. 

[1m ago](https://chat.canonical.com/canonical/pl/hn8zjdap6pnxzyg9smxat1seto)
In the future if we split the client and api packages into their own repo I would expect us to use a v1 go mod version and then we can do release tags, but I still doubt we would have a separate LTS series for that repo.
```